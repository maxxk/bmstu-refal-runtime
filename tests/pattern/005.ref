$ENTRY Go
{
	= <Prout <RollBackFunc a b c d>>
	  <Prout <NoRollBackFunc a b c d>>
}

RollBackFunc
{
	e.1 e.2 : e.1 c d = Success;
}

NoRollBackFunc
{
	e.1 e.2 :: e.1 c d = Success;
}

