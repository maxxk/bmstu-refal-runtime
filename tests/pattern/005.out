Success 
[VM]: Func call failed!
